src/App.css

```diff
@@ -81,7 +84,7 @@ li {
   list-style: none;
 }
 
-input[type='email'],
+/* input[type='email'],
```

Comment:
It would be better to remove the lines completely than comment them out if they are not being used anymore. I find that that ends up happening anyway over time, and we always have git blame to find the old code. Let me know if you had something else in mind.